srcpkgs/pekwm/template

```diff
@@ -1,33 +1,17 @@
 # Template file for 'pekwm'
 pkgname=pekwm
-version=0.1.17
-revision=4
-build_style=gnu-configure
-configure_args="--with-pkg-exec-prefix=/usr
- ac_cv_lib_X11_XOpenDisplay=yes ac_cv_lib_Xft_XftFontOpen=yes
- ac_cv_lib_Xrender_XRenderCreatePicture=yes ac_cv_lib_Xpm_XpmReadFileToPixmap=yes
- ac_cv_lib_Xinerama_XineramaQueryScreens=yes ac_cv_lib_Xext_XShapeCombineShape=yes
- ac_cv_lib_Xrandr_XRRQueryExtension=yes ac_cv_lib_fribidi_fribidi_version_info=yes"
+version=0.1.18
+revision=1
+build_style=cmake
+makedepends="libjpeg-turbo-devel libpng-devel libXinerama-devel libSM-devel
+ libXft-devel libXrender-devel fontconfig-devel libXpm-devel libXrandr-devel"
 short_desc="Window manager based on aewm++"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2"
+maintainer="biopsin <biopsin@teknik.io>"
+license="GPL-2.0-or-later"
 homepage="https://www.pekwm.org/projects/pekwm"
-distfiles="${DEBIAN_SITE}/main/p/pekwm/pekwm_${version}.orig.tar.bz2"
-checksum=8a1fd3bf9f38e8c7bb2b2864c090f986b60cec2281ecf1bba462d120fb327d00
+distfiles="https://github.com/pekdon/pekwm/releases/download/release-${version}/pekwm-${version}.tar.gz"
+checksum=7becc43b79eab0ff754572852e385435222420385107c881a93ac4dc4ae96065
 
-hostmakedepends="pkg-config"
-makedepends="libjpeg-turbo-devel libpng-devel libXinerama-devel
- libSM-devel libXft-devel libXrender-devel fontconfig-devel libXpm-devel libXrandr-devel"
-conf_files="
-	/etc/pekwm/vars
-	/etc/pekwm/mouse
-	/etc/pekwm/menu
-	/etc/pekwm/start
-	/etc/pekwm/keys
-	/etc/pekwm/autoproperties_typerules
-	/etc/pekwm/config
-	/etc/pekwm/autoproperties
-	/etc/pekwm/mouse_system
-	/etc/pekwm/mouse_sloppy
-	/etc/pekwm/mouse_click
```

Comment:
Did it stop using these files?

Reply:
No they are still in use. 
Hm, ok I see what the conf_files purpose is (pkg ownership),
I shall correct my blunder in a short while.